app/views/employees/index.html.erb

```diff
@@ -1,6 +1,9 @@
-<p id="notice"><%= notice %></p>
 
-<h1>Employees</h1>
+<%= content_for :title do %>
+    <h1 class="text-primary text-2xl text-center">Employees</h1>
```

Comment:
check indentation

Reply:
The indentation spaces were modified in this file